index.js

```diff
@@ -0,0 +1,88 @@
+const express = require('express');
+const bcrypt = require('bcryptjs');
+const db = require('./data/dbConfig.js');
+const session = require('express-session');
+const server = express();
+
+const PORT = 4200;
+
+server.use(express.json());
+
+server.use(session({
+    name: 'sessionTest',
+    secret: 'SECRET',
```

Comment:
I would not suggest using such an obvious secret. It should be something that should not be easily hacked